src/pages/docs/[[...filePath]].tsx

```diff
@@ -0,0 +1,173 @@
+import renderToString from "next-mdx-remote/render-to-string";
```

Comment:
Please try to do a git mv in this case to avoid recreating a file (here that's too late I don't think you can fix it but that's ok, that's minor)